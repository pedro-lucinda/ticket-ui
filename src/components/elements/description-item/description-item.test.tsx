import { render } from '@testing-library/react'
import React from 'react'
import { DescriptionItem } from '.'

describe('Description Item', () => {
  it('renders the description item correctly', () => {
    const mockProps = {
      content: 'Staples Center, Los Angeles, CA, US',
      label: 'Location',
    }

    const { container } = render(<DescriptionItem {...mockProps} />)
    expect(container).toMatchSnapshot()
  })
})
