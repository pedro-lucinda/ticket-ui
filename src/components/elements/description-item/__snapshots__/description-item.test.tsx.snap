// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Item renders the description item correctly 1`] = `
<div>
  <div
    class="flex md:gap-5 gap-1 max-md:flex-wrap"
  >
    <p
      class="font-bold text-sm capitalize"
    >
      Location
      :
    </p>
    <p
      class="font-medium text-sm"
    >
      Staples Center, Los Angeles, CA, US
    </p>
  </div>
</div>
`;
